taf/api/repository.py

```diff
@@ -188,26 +192,27 @@ def create_repository(
         roles_key_infos, keystore
     )
 
+    roles_keys_data = from_dict(roles_key_infos, RolesKeysData)
```

Comment:
Could we add unit tests for loading in `keys-description.json`? I assume we can create a couple of `keys-description.json` fixtures and assert expected states of the model? Looks like we don't need roundtrip tests, since it seems like we're doing json -> model.

Reply:
Yes. I can add unit tests for this without testing create repository and add roles, just the json-> model transformation